src/pages/dashboard/_StateCumulativeTestsContainer.js

```diff
@@ -191,20 +191,25 @@ export default function CumulativeTestsByStateContainer() {
       </p>
       <h3 className={dashboardStyles.chartTitle}>Cumulative tests by state</h3>
       <div className="chart-header">
-        <div
-          className="dashboard-toggle"
-          onClick={toggleChartData}
-          onKeyPress={toggleChartData}
-          role="switch"
-          aria-checked={useTestsPerCapita}
-          tabIndex={0}
-        >
-          <span className={useTestsPerCapita ? '' : 'active'}>Total Tests</span>
-          <span className={useTestsPerCapita ? 'active' : ''}>
-            Tests Per Capita
-          </span>
+        <div className="dashboard-toggle-wrapper">
+          <div className="dashboard-toggle-label">Display by:</div>
```

Comment:
This layout breaks on small screens. This is at iPhone X dimensions:

<img width="374" alt="Screen Shot 2020-04-18 at 11 27 52 AM" src="https://user-images.githubusercontent.com/63169602/79641811-c2eabe00-8167-11ea-8fbf-a0851f24cc85.png">

Reply:
@vijaysharwar thank you, just pushed an update for this (below screenshot at iPhone X dimensions):

![image](https://user-images.githubusercontent.com/1994207/79642446-7ef9b800-816b-11ea-97b2-7bb3a47c714f.png)